sentinelhub/geopedia.py

```diff
@@ -248,3 +248,12 @@ def get_field_iterator(self, field):
         """
         for feature in self:
             yield feature['properties'].get(field, [])
+
```

Comment:
Would it maybe make more sense to add this parameter to `__init__`? That is because once you start iterating and making HTTP requests it doesn't make much sense to change query parameter somewhere in the middle.